docs/api/ethereum.md

```diff
@@ -1384,7 +1384,8 @@ const mutation = await ledger.grantAbilities(accountId, abilities);
 
 **See also:**
 
-[revokeAbilities](#revoke-abilities)
+[Ledger abilities](#ledger-abilities)
```

Comment:
small case for `ledger`

Reply:
If you check here: https://docs.0xcert.org/api/ethereum.html#ledger-abilities The title is upper case.